modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-template/en-US.html

```diff
@@ -11,8 +11,8 @@
 
 			<p>Please review the request at the following link:</p>
 
-			<a href="[$PORTAL_URL$]/[$GROUP_FRIENDLY_URL$]/l/[%MDFREQUEST_ID%]">
-				[$PORTAL_URL$]/[$GROUP_FRIENDLY_URL$]/l/[%MDFREQUEST_ID%]
+		<a href="[$PORTAL_URL$]/web[$GROUP_FRIENDLY_URL$]/l/[$MDFREQUEST_ID$]">
+				[$PORTAL_URL$]/web[$GROUP_FRIENDLY_URL$]/l/[$MDFREQUEST_ID$]
```

Comment:
```suggestion
		<a href="[$PORTAL_URL$]/web[$GROUP_FRIENDLY_URL$]/l/[%MDFREQUEST_ID%]">
				[$PORTAL_URL$]/web[$GROUP_FRIENDLY_URL$]/l/[%MDFREQUEST_ID%]
```

Reply:
Pay attention because they are different placeholders to be replaced.

With `$` is site-initializer's replace.
With `%` is notification template term replace.

Reply:
Nice, done!